public/docs/ts/latest/tutorial/toh-pt1.jade

```diff
@@ -11,19 +11,20 @@ include ../_util-fns
 .filetree
   .file angular-tour-of-heroes
   .children
+    .file node_modules ... 
+    .file package.json
```

Comment:
This would not appear here in an OS file listing which shows all folders/directories first followed by files.

OTOH, what really matters from a developer perspective is the `src` folder.

Finally, in a scrub of the tutorial "coming soon" we will eliminate all of these file trees because the code itself tells the story.

Therefore, I'd rather not pursue this change at this time although I thank you for your readiness to help.